{"name": "测试用例描述：错误的密码登录失败", "status": "passed", "description": "测试描述：错误的密码登录失败", "steps": [{"name": "输入正确账号", "status": "passed", "attachments": [{"name": "正确账号", "source": "7cc3b205-efe0-4fc6-bed9-23b5299629c6-attachment.attach"}], "start": 1651818994891, "stop": 1651818994891}, {"name": "输入错误密码", "status": "passed", "attachments": [{"name": "错误密码", "source": "c7aecc9d-8f73-416c-bd5a-a22732f7b189-attachment.attach"}], "start": 1651818994931, "stop": 1651818994931}, {"name": "输入正确验证码", "status": "passed", "attachments": [{"name": "正确验证码", "source": "281b1453-2dc3-4d1e-adcc-c492a562ded8-attachment.attach"}], "start": 1651818994977, "stop": 1651818994978}], "attachments": [{"name": "截图", "source": "6f9b2e65-c4ad-4f39-82a7-eeae5f4cd84f-attachment.png", "type": "image/png"}], "start": 1651818994890, "stop": 1651818997319, "uuid": "d10b8ec0-1322-428e-a886-082266b3eb64", "historyId": "8ec91ebb07cb2b45a38478eca1be69f5", "testCaseId": "a86650c2206e92995fa9f8bad029494f", "fullName": "test_two.Test#test_06", "labels": [{"name": "story", "value": "登录"}, {"name": "epic", "value": "xxx测试"}, {"name": "feature", "value": "登录模块"}, {"name": "severity", "value": "critical"}, {"name": "suite", "value": "test_two"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "9800-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_two"}]}